beacon-chain/rpc/eth/events/events_test.go

```diff
@@ -78,20 +81,26 @@ func requireAllEventsReceived(t *testing.T, stn, opn *mockChain.EventFeedWrapper
 		}
 	}()
 	select {
+	case entry := <-logs:
+		errAttr, ok := entry.Data[logrus.ErrorKey]
+		if ok {
```

Comment:
This ok naming convention is pretty confusing like normally we do  
`if !ok {
   t.Errorf("unexpected error  in logs: %v")
}`
rather name it `err` just a suggestion

Reply:
`ok` is a conventional variable name in go for receiving the bool that indicates whether a key in a dict exists, for example: https://go.dev/tour/moretypes/22